web/packages/teleport/src/Locks/useGetTargetData.tsx

```diff
@@ -29,6 +29,7 @@ export const lockTargets: LockTarget[] = [
   { label: 'Windows Desktop', value: 'windows_desktop' },
   // Skipped for now because it's not fully implemented.
   // { label: 'Device', value: 'device' },
+  // TODO(sshahcodes): add support for locking devices
```

Comment:
FYI @sshahcodes - I hope you don't mind, but I thought this would be a nice thing to put on your todo list since you added device listing to the web UI :-)

Reply:
Sounds good 👍